bin/create_charts.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
this is a large-ish change. we dont know for e,g if all the other driver work with python3.